### ISSUE TITLE
Vendor is not specified in RxIssueRegistry

### ISSUE THREAD

**Issue by siempredelao** [littlerobots/rxlint#9]
Hey folks!

When I run the lint checks in my project, I see the following report on my console:
```
nl.littlerobots.rxlint.RxIssueRegistry in /home/david/.gradle/caches/transforms-3/5e006e2dd79821b780f390d44e831beb/transformed/rxlint-1.7.7/jars/lint.jar does not specify a vendor; see IssueRegistry#vendor
```

Could you please specify the vendor, so this warning is removed? Nothing urgent, this is just a "warning", the build finishes without errors.

**Comment by hvisser** [comment]
What version of AS are you seeing this? The AGP dependencies should probably be bumped again to the latest stable version for this.

**Comment by siempredelao** [comment]
I'm on _Android Studio Arctic Fox | 2020.3.1 Patch 2_.

**Comment by hvisser** [comment]
@siempredelao would you be able to test if the warning is gone with the `1.7.8-SNAPSHOT` I just uploaded? You'd need to add `https://oss.sonatype.org/content/repositories/snapshots/` as a Maven repository for that.

**Comment by hvisser** [comment]
Assuming that this worked, version 1.7.8 implements this change.
